### ISSUE TITLE
Modify Home page to add "Recently added SONGS (listed by newest first, up to a year's worth)

### ISSUE THREAD

**Issue by etyrnal** [airsonic/airsonic#843]
## Problem description

The problem is Airsonic is so completely ALBUM-centric, and ARTIST-centric, that collections of music, and other audio, that do NOT fit neatly into Artists, album hierarchical format, cannot be intelligently accessed.  The possibility needs to be added to search for these either by name/filename/id3 tag content/filename such that when they come up in he search results they can be selected, played, added to playlist, added to play queue next/last, starred, rated, shared, etc. 

### Steps to reproduce

Attempt to locate disenfranchised, non-hierarchically-organized song/track/audio-file (i.e. just single, lonely mp3 file by name, for example an audio recording of a street busker made with a cellphone)  using search (we already know you cannot BROWSE to it), and try to DO anything with it if it's in a folder along with THOUSANDS of others.  Not possible.

**Where in the source do i find the code that generates the home page header/frame?**

**Comment by muff1nman** [comment]
Yes this is a known issue and I think further solidifies the comment I made here: https://github.com/airsonic/airsonic/pull/567#issuecomment-338539643

**Comment by etyrnal** [comment]
Is this idea addressed, or potentially being addressed by some progress goal?  Just wondering if there are any developments that will address this at some point?

**Comment by etyrnal** [comment]
nooooO!  Please no staleness.  This is the whole reason  my usage of airsonic has drastically dwindled.  It's the only  workaround to not being able to browse songs.  

Not every music appreciator listens to music by the album, or by the artist.

I add single tracks to my library.  I'd like to be able to pull up a list of recently added SONGS, newest to oldest for a specified date range -- defaulting to a year tho.

These are the tracks i listen to most often.

There's not good way of doing this.

**Comment by VirsabiMunk** [comment]
+1
